nntrainer/graph/network_graph.cpp

```diff
@@ -37,35 +38,68 @@ namespace nntrainer {
 static const std::vector<std::string> in_place_layers = {
   ActivationLayer::type, BatchNormalizationLayer::type};
 
+int NetworkGraph::compile(const LossType loss_type) {
```

Comment:
c and c++ api has compile. So.. API is still in neuralnet and it it wrapper of this?

Reply:
`NeuralNetwork::Compile` still exists, and it is just a wrapper for this.
In later stages, once `NeuralNetwork::Compile()` will decide if `NetworkGraph::Compile` must be done or it is already done offline etc.